packages/widgets/src/calendar/calender-day.tsx

```diff
@@ -59,20 +64,19 @@ export const CalendarDay = ({ date, events, disabled }: CalendarDayProps) => {
   );
 };
 
-interface DayIndicatorProps {
-  size: number;
-  color: string;
+interface ArrIndicatorProps {
   events: CalendarEvent[];
-  children: ReactNode;
-  position: IndicatorProps["position"];
 }
 
-const DayIndicator = ({ size, color, events, children, position }: DayIndicatorProps) => {
-  if (events.length === 0) return children;
-
+const NotificationIndicator = ({ events }: ArrIndicatorProps) => {
+  const notificationEvents = [...new Set(events.map((event) => event.links[0]?.notificationColor))].filter(String);
```

Comment:
Will there always be at least one link? I doubt

Reply:
No, and that will create an undefined which will get filtered out